code/modules/mob/living/basic/lavaland/mook/mook.dm

```diff
@@ -68,6 +66,14 @@
 
 	AddComponent(/datum/component/obeys_commands, pet_commands)
 
+/// Returns a list of actions and blackboard keys to pass into `grant_multiple_actions`.
+/// Uses the dot operator explicitly so it's more comphrensible on subtypes.
```

Comment:
How is the dot operator any more explicit than just doing `return list()`?

Reply:
this is some old code that i forgot to remove the comment on